server/meshmodel/k8svault-controller/relationships/edge_relationship.json

```diff
@@ -0,0 +1,61 @@
+{
+  "apiVersion": "vault.infra.doodle.com/v1beta1",
+  "kind": "Edge",
+  "metadata": {
+    "description": ""
+  "model": {
+    "name": "k8svault-controller",
+    "version": "0.1.2",
+    "displayName": "K8s Vault Controller",
+    "category": {
+      "name": "Security \u0026 Compliance",
+      "metadata": null
+    },
+    "metadata": {}
+  },
+   "subType": "Mount",
```

Comment:
Could you please ensure that you are specifying the relationship for subType -> Mount? Currently, I don't see any components being mounted.